source/agora/cli/gen/main.d

```diff
@@ -0,0 +1,200 @@
+/*******************************************************************************
+
+    Entry point for the agora-gen
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.cli.gen.main;
```

Comment:
`gen` is a bit too generic of a name. Something like `check-vtable` would be more accurate.